src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -3,50 +3,50 @@
     "IncludeScopes": false,
     "Debug": {
       "LogLevel": {
-        "Default": ""
+        "Default": "Warning"
       }
     },
     "Console": {
       "LogLevel": {
-        "Default": ""
+        "Default": "Warning"
       }
     }
   },
   "CompilerHost": {
-    "CompilerHostBinaryLocation": "",
+    "CompilerHostBinaryLocation": "..\\Diagnostics.CompilerHost\\bin\\Debug\\netcoreapp2.0",
     "CompilerHostPort": "1900",
     "PollingIntervalInSeconds": "60",
     "ProcessMemoryThresholdInMB": "100"
   },
   "SourceWatcher": {
-    "WatcherType": "",
+    "WatcherType": "1",
     "PollingIntervalInSeconds": "10",
     "Local": {
       "LocalScriptsPath": ""
     },
     "Github": {
-      "DestinationScriptsPath": "",
+      "DestinationScriptsPath": ".\\bin\\Detectors",
       "UserName": "",
-      "RepoName": "",
-      "Branch": "",
+      "RepoName": "detectorslist",
+      "Branch": "master",
       "AccessToken": ""
     }
   },
-  "Kusto": {
-    "ClientId": "",
-    "AppKey": "",
-    "DBName": "",
-    "KustoRegionGroupings": "",
-    "KustoClusterNameGroupings": "",
-    "KustoClusterFailoverGroupings": "",
-    "HeartBeatConsecutiveFailureLimit": 5,
-    "HeartBeatConsecutiveSuccessLimit": 5,
-    "HeartBeatQuery": "",
-    "HeartBeatTimeOutInSeconds": 10,
-    "HeartBeatDelayInSeconds": 20,
-    "AADAuthority": "https://login.windows.net/microsoft.com",
-    "AADKustoResource": "https://wawskustotest.kusto.windows.net"
-  },
+"Kusto": {
+  "ClientId": "",
+  "AppKey": "",
+  "DBName": "wawsprod",
+  "KustoRegionGroupings": "BAY:MWH:MSFTBAY,BLU:BN1:YQ1:MSFTBLU,CH1:CQ1:CY4:DM1:SN1:YT1,AM2,DB3:MSFTDB3,*",
```

Comment:
Lets keep this appsettings file clean, just schema and not values..

Reply:
I understand but I think changes in app settings are worth it to allow developers to quickly get up and running without having to pass a appsettings.developer.json file around. If you think it should be removed I will do that.

Reply:
Ya, earlier we used to have this but we decided to keep the app setting file clear in order to not leak the secrets accidentally. In general, I feel its a good practice to keep the app settings clean and have environment specific overrides.

With this PR, developers wont even need appsettings.developer.json file right?

Reply:
The aim is to not need a appsettings.developer.json file but if we keep app setting file clear then we will need it. I want to check in at the least an example app settings that when used  should not require anything new.